src/BloomExe/Publish/MakePdfUsingGeckofxHtmlToPdfProgram.cs

```diff
@@ -52,6 +57,14 @@ class MakePdfUsingGeckofxHtmlToPdfProgram
 			{
 				exePath = Path.Combine(execDir, "GeckofxHtmlToPdf.exe");
 			}
+			if (!File.Exists(exePath))
+			{
+				var msg = LocalizationManager.GetString("InstallProblem.GeckofxHtmlToPdf",
+					"A component of Bloom, GeckofxHtmlToPdf.exe, seems to be missing. This prevents previews and printing. Antivirus software sometimes does this. You may need technical help to repair the Bloom installation and protect this file from being deleted again.");
+				var caption = LocalizationManager.GetString("InstallProblem.MissingFile", "Missing file");
+				MessageBox.Show(null, msg, caption, MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
Please use a yellow box so that it is easy for people to report the problem to us.